sys/include/net/gnrc/netif/hdr.h

```diff
@@ -199,6 +199,23 @@ gnrc_pktsnip_t *gnrc_netif_hdr_build(uint8_t *src, uint8_t src_len, uint8_t *dst
  */
 void gnrc_netif_hdr_print(gnrc_netif_hdr_t *hdr);
 
+/* @brief   Check if packet is for broadcast
+ *
+ * @param[in]   pkt     packet to check
+ *
+ * @return              true if the packet is for broadcast, otherwise false
+ */
+bool gnrc_netif_hdr_chk_pkt_bcast(gnrc_pktsnip_t* pkt);
```

Comment:
@miri64 Do you think I should restrict the input to `gnrc_netif_hdr_t *`, but not for `gnrc_pktsnip_t*`